CHANGELOG.md

```diff
@@ -1,6 +1,50 @@
 Change log
 ==========
 
+1.7.1 (2016-05-04)
+-----------------
+
+Bug Fixes
+
+- Fixed a bug where the output of `docker-compose config` for v1 files
+  would be an invalid configuration file.
+
+- Fixed a bug where `docker-compose config` would not check the validity
+  of links.
+
+- Fixed an issue where `docker-compose help` would not output a list of
+  available commands and generic options as expected.
+
+- Fixed an issue where filtering by service when using `docker-compose logs`
+  would not apply for newly created services.
+
+- Fixed a bug where unchanged services would sometimes be recreated in
+  in the up phase when using Compose with Python 3.
+
+- Fixed an issue where API errors encountered during the up phase would
+  not be recognized as a failure state by Compose.
+
+- Fixed an issue with Windows daemons where compose would try to disconnect
+  containers from their networks despite that feature not being available.
```

Comment:
It turns out this didn't actually fix the issue because the windows daemon doesn't use the same network driver.  We should probably drop this from the release notes.  Sorry I didn't communicate this earlier.